themes/default/content/docs/guides/pulumi-packages/how-to-author.md

```diff
@@ -116,3 +131,9 @@ From there, a Pulumi employee will work with you to get your Pulumi Package publ
 ## Congratulations
 
 Congratulations on publishing your Pulumi Package!
+
+[^1]: Pulumi packages consist of SDKs, as well as a binary to facilitate the actual task (creating cloud resources, ect). Package
+    managers (`npm`, `nuget`, `pip`, `github`) host the SDKs, but we need to know where the plugin is hosted. When a package
```

Comment:
Since these aren't code or commands here (but rather refer to the package managers themselves):

```suggestion
    managers (npm, NuGet, Pip, GitHub) host the SDKs, but we need to know where the plugin is hosted. When a package
```